content/spin/v2/key-value-store-tutorial.md

```diff
@@ -298,22 +293,29 @@ export const handleRequest: HandleRequest = async function (request: HttpRequest
   switch (request.method) {
     case "POST":
       store.set(request.uri, request.body || (new Uint8Array()).buffer)
+      console.log(`Storing value in the KV store with ${request.uri} as the key`);
       break;
     case "GET":
       let val
       try {
         val = store.get(request.uri)
         body = decoder.decode(val)
+      	console.log(`Found value for the key ${request.uri}`);
       } catch (error) {
+      	console.log(`Key ${request.uri} not found`);
         status = 404
       }
       break;
     case "DELETE":
       store.delete(request.uri)
+      console.log(`Deleted Key ${request.uri}`);
       break;
     case "HEAD":
       if (!store.exists(request.uri)) {
```

Comment:
The "if not .. else" can be hard for human readers to parse.  If we are adding an `else` then it would be good to flip this to "if exists { log found} else { log not found; status = 404 }"

Reply:
👍🏻